config/config.exs

```diff
@@ -168,6 +168,14 @@ config :surface, :components, [
 config :oli, :privacy_policies,
   url: System.get_env("PRIVACY_POLICIES_URL", "https://www.cmu.edu/legal/privacy-notice.html")
 
+# Configure footer text and links
+config :oli, :footer,
+  text: System.get_env("FOOTER_TEXT", "By accessing this site, you agree to the "),
+  link_1_location: System.get_env("FOOTER_LINK_1_LOCATION", "https://www.argos.education/terms"),
```

Comment:
The default values for link locations should not be Argos related, but rather should be CMU OLI links.

Reply:
Okay makes sense @darrensiegel ! What values do you propose? Or should we leave them empty?

Reply:
We should make them default to empty, as Proton currently does not have them populated